test/runtests.jl

```diff
@@ -5,12 +5,21 @@ using TimeSpans, Dates, Onda, Intervals
 using JSON3, Arrow
 using Aqua
 
+@static if isdefined(Onda, :SignalV2)
+    const _SamplesInfo = SamplesInfoV2
+    sensor_type_name = :sensor_type
+else
+    const _SamplesInfo = SamplesInfo
+    sensor_type_name = :kind
```

Comment:
Don't need to change the casing but should at least make these `const`
```suggestion
    const SENSOR_TYPE_NAME = :sensor_type
else
    const _SamplesInfo = SamplesInfo
    const SENSOR_TYPE_NAME = :kind
```